### ISSUE TITLE
a11y CRITICAL: `auro-combobox` `#trigger` has empty node of aria-labelledby

### ISSUE THREAD

**Issue by sun-mota** [AlaskaAirlines/auro-formkit#614]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [ ] Desktop
- [ ] Mobile

### Steps to reproduce

```
      Error: Accessibility Violations
      ---
      Rule: aria-input-field-name
      Impact: serious
      ARIA input fields must have an accessible name (https://dequeuniversity.com/rules/axe/4.10/aria-input-field-name?application=axeAPI)
      
      Issue target: auro-combobox,#e6aae64b-c1d4-41a8-ad66-9e9aaaf9b8a2,#trigger
      Context: <div id="trigger" class="trigger" part="trigger" showborder="" tabindex="0" role="combobox" aria-expanded="false" aria-controls="e6aae64b-c1d4-41a8-ad66-9e9aaaf9b8a2" aria-labelledby="triggerLabel" style="height: 56px;">
      Fix any of the following:
        aria-label attribute does not exist or is empty
        aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
        Element has no title attribute
```

### Reproducible on the Auro doc site

- [ ] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).
